doc/source/conf.py

```diff
@@ -317,3 +318,37 @@ def prepare_jinja_env(jinja_env) -> None:
     # Python std lib errors
     (r"py:obj", r"logging.PercentStyle"),
 ]
+
+
+def convert_notebooks_to_scripts(app: sphinx.application.Sphinx, doctree, docname):
+    """
+    Convert notebooks to scripts.
+
+    Parameters
+    ----------
+    app : sphinx.application.Sphinx
+        Sphinx application instance containing all the doc build configuration.
+    doctree : sphinx.util.nodes.Node
+        The document tree.
+    docname : str
+        The document name.
```

Comment:
```suggestion
        Sphinx instance containing all the configuration for the documentation build.
    doctree : sphinx.util.nodes.Node
        Sphinx document tree.
    docname : str
        Sphinx document name.
```
We want to avoid abbreviations, even "doc" in doc strings. Should the method description be: "Convert Jupyter notebooks to scripts." I was thinking this more descriptive would be helpful if conversions of other types of files were added later.